Mathlib/Probability/Moments.lean

```diff
@@ -261,6 +262,7 @@ theorem iIndepFun.integrable_exp_mul_sum [IsFiniteMeasure μ] {X : ι → Ω →
     refine IndepFun.integrable_exp_mul_add ?_ (h_int i (mem_insert_self _ _)) h_rec
     exact (h_indep.indepFun_finset_sum_of_not_mem h_meas hi_notin_s).symm
 
+-- TODO(vilin97): weaken `h_meas` to `AEMeasurable (X i)` or `AEStronglyMeasurable (X i)` throughout
```

Comment:
Are you planning on doing this refactor yourself in the near future? If not, you could open an issue and link it here

Reply:
I can try, but probably not in the next two weeks. I made an issue: https://github.com/leanprover-community/mathlib4/issues/20367.